LICENSE

```diff
@@ -1,4 +1,5 @@
 `createsamples.pl` and `mergevec.cpp`: Copyright (c) 2008, Naotoshi Seo
+`mergevec.py`: Copyright (c) 2014, Blake Wulfe
```

Comment:
Could you move this line two down, so the "From: ..." is below "createsamples.pl ..." line? Maybe you could also add a URL pointing to the mergevec.py repository